client/README.md

```diff
@@ -0,0 +1,77 @@
+# SDK Client
+
+## Instruction
+
+### Remove remaining data(if exists)
+
+```bash
+> rm -r ~/.gaiad
+> rm -r ~/.gaiacli
+```
+
+### Installation
+
+```bash
+> go get -u github.com/cosmos/cosmos-sdk
+package github.com/cosmos/cosmos-sdk: no Go files in /home/mossid/go/src/github.com/cosmos/cosmos-sdk
+> cd $GOPATH/src/github.com/cosmos/cosmos-sdk
+> git checkout develop
+Branch develop set up to track remote branch develop from origin.
+Switched to a new branch 'develop'
+> dep ensure
+> make install
+go install -ldflags "-X github.com/cosmos/cosmos-sdk/version.GitCommit=be7ec5b" ./cmd/gaia/cmd/gaiad
+go install -ldflags "-X github.com/cosmos/cosmos-sdk/version.GitCommit=be7ec5b" ./cmd/gaia/cmd/gaiacli
+```
+
+### Run node and server
+
+```bash
+# If your are going to connect to an external node(e.g. testnet), skip gaiad init and gaiad start
```

Comment:
Move this out of the bash window. Rephrase to "If you are going to connect to a full node on another machine (e.g. testnet), skip this part"